src/components/SchedulePage/Timeslot.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import Divider from '@material-ui/core/Divider';
+import Event from '../SchedulePage/Event';
+import ScheduleRoundedIcon from '@material-ui/icons/ScheduleRounded';
+
+const useStyles = makeStyles(theme => ({
+	time: {
+		fontFamily: theme.typography.fontFamily,
+		marginLeft: 4
+	},
+	timeContainer: {
+		display: 'flex',
+		alignItems: 'center'
+	}
+}));
+
+const formatTime = time => {
+	let hours = time.getHours();
+	let minutes = time.getMinutes();
+	const ampm = hours >= 12 ? 'PM' : 'AM';
+	hours %= 12;
+	hours = hours ? hours : 12; // the hour '0' should be '12'
+	minutes = minutes < 10 ? '0' + minutes : minutes;
+	const strTime = hours + ':' + minutes + ' ' + ampm;
+	return strTime;
+};
+
+function Timeslot({ time, events }) {
+	const classes = useStyles();
+	return (
+		<Container maxWidth="md">
+			<Grid container spacing={1}>
+				<Grid item xs={12}>
```

Comment:
I think it would be best if you find a way to avoid the use of `Grid`. Grid is designed to be primarily used for horizontal layout, like if we have several elements side-by-side. I have a suspicion that simply removing the two `Grid` elements would work just fine.

Additionally, the `Container` part should go in the `Schedule` component, surrounding both the title and this class. A rule of thumb is that you generally don't need have a `Container` inside another `Container`. If you want the `<Event>` to have a left margin, add one manually with CSS rather than relying on `Container`, which can contribute to inconsistencies like the following.

Desktop:

![image](https://user-images.githubusercontent.com/1538624/85486585-e3d0f480-b598-11ea-86a0-87651c3f66a3.png)

Mobile:

![image](https://user-images.githubusercontent.com/1538624/85486559-d61b6f00-b598-11ea-9a4c-1d1a126269b1.png)

Reply:
@TimothyGu I'm confused how I would get them next to each other when the screen is large if we remove the Grid.

Reply:
I'm not asking you to remove the Grid in Event, where I assume it's necessary as you said to get the event name and description in horizontal layout. I'm saying that you probably don't need it in this file, Timeslot.js.

Reply:
To summarize:

In `Timeslot.js`, delete `Container` and `Grid`.

In `Event.js`, delete `Container`.

In `Schedule.js`, add `Container`.